runners/flink/src/test/java/org/apache/beam/runners/flink/FlinkSavepointTest.java

```diff
@@ -191,7 +193,7 @@ private void runSavepointAndRestore(boolean isPortablePipeline) throws Exception
   private JobID executeLegacy(Pipeline pipeline) throws Exception {
     JobGraph jobGraph = getJobGraph(pipeline);
     flinkCluster.submitJob(jobGraph).get();
-    return waitForJobToBeReady();
+    return jobGraph.getJobID();
```

Comment:
Why should we revert this line? I think it was a good change, even if it didn't fix the flake.

Reply:
@benWize - Could you respond to this comment?

Reply:
Yes, I agree it is not necessary to revert this line, so I pushed a new commit.